spec/controllers/tasks_controller_spec.rb

```diff
@@ -16,7 +16,7 @@
   let(:next_week) { Time.zone.today + 7.days }
 
   before do
-    stub_sso_pom_data(prison)
+    stub_sso_pom_data(prison, username)
```

Comment:
now I'm slightly confused - I think we should only have 1 'login as POM' method but we seem to have 2 (stub_sso_pom_data and stub_signed_in_pom)